+ Tesst Failure reason:
+[    0.148455] Out of memory: Killed process 36 (host0) total-vm:4kB, anon-rss:0kB, file-rss:0kB, shmem-rss:0kB
+[    0.148455] Kernel panic - not syncing: System is deadlocked on memory
+[    0.149001] ---[ end Kernel panic - not syncing: System is deadlocked on memory ]---
+[[  SGX-LKL ]] FAIL: Kernel panic! Run DEBUG build with SGXLKL_KERNEL_VERBOSE=1 for more information. Aborting...
+2020-07-24T14:59:18.000000Z [(H)ERROR] tid(0x7fa372ffd700) | :OE_ENCLAVE_ABORTING [/mnt/vamsi/repo/sgx-lkl/openenclave/host/calls.c:oe_call_enclave_function_by_table_id:91]
+[   SGX-LKL  ] ethread (6: 19) [   SGX-LKL  ] FAIL: sgxlkl_ethread_init() failed (id=6 result=19 (OE_ENCLAVE_ABORTING))

+ Patch Description:
+ Tests were failing with Kernel panic (out of memory) in loop device.
+ loop device filesystem is having a limit of 32MB
+ So modified the tests using root filesystem.
diff --git a/testcases/kernel/syscalls/fallocate/fallocate04.c b/testcases/kernel/syscalls/fallocate/fallocate04.c
index 936d0d1b0..4cacadd4f 100644
--- a/testcases/kernel/syscalls/fallocate/fallocate04.c
+++ b/testcases/kernel/syscalls/fallocate/fallocate04.c
@@ -285,6 +285,7 @@ static void run(unsigned int i)

 static void setup(void)
 {
+       SAFE_MKDIR(MNTPOINT, 0644);
        fd = SAFE_OPEN(FNAME, O_RDWR | O_CREAT, 0700);

        get_blocksize();
@@ -294,6 +295,8 @@ static void cleanup(void)
 {
        if (fd > 0)
                SAFE_CLOSE(fd);
+       remove(FNAME);
+       SAFE_RMDIR(MNTPOINT);
 }

 static struct tst_option opts[] = {
@@ -307,9 +310,5 @@ static struct tst_test test = {
        .setup = setup,
        .test = run,
        .tcnt = ARRAY_SIZE(tcases),
-       .mount_device = 1,
-       .mntpoint = MNTPOINT,
-       .all_filesystems = 1,
-       .needs_tmpdir = 1,
        .needs_root = 1,
 };

